borrow-collateral-ratio/src/agent.js

```diff
@@ -0,0 +1,146 @@
+const {
+  Finding, FindingSeverity, FindingType, ethers, getEthersProvider,
+} = require('forta-agent');
+const BigNumber = require('bignumber.js');
+
+const { getAbi } = require('./utils');
+
+// load any bot configuration parameters
+const config = require('../bot-config.json');
+
+// set up a variable to hold initialization data used in the handler
+const initializeData = {};
+const DECIMALS_ABI = ['function decimals() view returns (uint)'];
+
+function createAlert(protocolName, protocolAbbrev, developerAbbrev, type, severity, metadata) {
+  const { assetTokenSymbol, assetTokenAddress, maxUtilizationRate } = metadata;
+  return Finding.fromObject({
+    name: `${protocolName} Borrow Collateral Ratio`,
+    description: 'The ratio of total borrow amount to total liquidity exceeds the configured '
```

Comment:
Elsewhere in this bot we use the word "collateral" but in this description we use the word "liquidity."